src/settings-ui/Settings.UI.Library/PowerOcrProperties.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json;
+
+namespace Microsoft.PowerToys.Settings.UI.Library
+{
+    public class PowerOcrProperties
+    {
+        public PowerOcrProperties()
+        {
+            ActivationShortcut = new HotkeySettings(true, false, false, true, 0x4F);
```

Comment:
activation shortcut settings needed in PowerOCR settings page

Reply:
Added